.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: "Community Discussion Forums"
+    url: "https://discourse.firez.one"
+    about: "Ask questions, get help from other Firezone users, and suggest features."
+  - name: "Community Slack"
```

On our pricing page we say "Community Support", maybe we should name it something like this for transparency?
```suggestion
  - name: "Community Support - Discussion Forums"
    url: "https://discourse.firez.one"
    about: "Ask questions, get help from other Firezone users, and suggest features."
  - name: "Community Support - Slack"
```